README.md

```diff
@@ -135,6 +157,33 @@ setupController: function(controller, model) {
   controller.set('products', model);
 }
 ```
+### event hooks
+
+ember-infinity provides a route mixin that enables us to hook into the following events:
+
+* **infinityModelUpdated**
+
+This gets fired up when data is loaded into the model.
+
+* **infinityModelLoaded**
+
+Fired up when there's no more data to load from the backend, needless to say this depends on your `meta.total_pages` key returned by your server.
```

Comment:
Same here, can we change this to:

Triggered on the route when the infinityModel is fully loaded.

**Args:**
- lastPageLoaded
- totalPages
- infinityModel